homeassistant/components/cloud/http_api.py

```diff
@@ -327,6 +328,32 @@ async def post(self, request: web.Request, data: dict[str, Any]) -> web.Response
         return self.json_message("ok")
 
 
+@websocket_api.websocket_command({vol.Required("type"): "cloud/remove_data"})
+@websocket_api.async_response
+async def websocket_cloud_remove_data(
+    hass: HomeAssistant,
+    connection: websocket_api.ActiveConnection,
+    msg: dict[str, Any],
+) -> None:
+    """Handle request for account info.
+
+    Async friendly.
+    """
+    cloud: Cloud[CloudClient] = hass.data[DOMAIN]
+    if cloud.is_logged_in:
```

Comment:
This is weird, right? Like, where would this button be? Can't we just have removal of data be a checkbox when you logout in the UI?

Reply:
That's would be a really weird UI, because you can then only clean up after successfully logging back in again.

Reply:
The frontend PR has removing data as a separate action so we can't include this in the logout command